Makefile

```diff
@@ -100,6 +100,13 @@ ifneq (${TRAVIS_TAG},)
 	$(call do-docker-retag-and-push-public,${TRAVIS_TAG})
 endif
 
+.PHONY: helm-push-public
+	DOCKER_HOSTNAME=$DOCKER_PUBLIC_HOSTNAME DOCKER_NAMESPACE=${DOCKER_PUBLIC_NAMESPACE} make -C modules helm-chart-push
+ifneq (${TRAVIS_TAG},)
+	DOCKER_HOSTNAME=$DOCKER_PUBLIC_HOSTNAME DOCKER_NAMESPACE=${DOCKER_PUBLIC_NAMESPACE} DOCKER_TAGNAME=${TRAVIS_TAG} make -C modules helm-chart-push
+endif
```

Comment:
1. you are missing the target `helm-push-public` but I would rather you just add the line:

```
DOCKER_HOSTNAME=$DOCKER_PUBLIC_HOSTNAME DOCKER_NAMESPACE=${DOCKER_PUBLIC_NAMESPACE} DOCKER_TAGNAME=$1 make -C modules helm-chart-push
```

to the end of macro:

```
do-docker-retag-and-push-public
```

Reply:
I added `helm-push-public`. Why do you suggest to join it with do-docker-retag-and-push-public?

Reply:
its' an optimization making code more elegant ... but can be merged as-is